crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -466,11 +466,7 @@ impl<F> TryFrom<&types::RefundsRouterData<F>> for AuthorizedotnetCreateSyncReque
     type Error = error_stack::Report<errors::ConnectorError>;
 
     fn try_from(item: &types::RefundsRouterData<F>) -> Result<Self, Self::Error> {
-        let transaction_id = item
-            .response
-            .as_ref()
-            .map(|refund_response_data| refund_response_data.connector_refund_id.clone())
-            .ok();
+        let transaction_id = item.request.connector_refund_id.clone();
```

Comment:
shouldn't we handle required field validation here?

Reply:
According to the earlier code, we are handling the existence of the ID somewhere ahead in the code. 
I have done an inplace change for the connector refund id, to avoid any unwanted bugs to arise because of this change.